### ISSUE TITLE
lint: add checks for uninformative comments

### ISSUE THREAD

**Issue by fexolm** [PieselBois/kfulint#24]
As @Quasilyte mentiond in #22 we could add checker for uninformative comments like so
``` // Foo ... ```

**Comment by fexolm** [comment]
@Quasilyte, @weeellz, @ludweeg do you have any idea how we should detect uninformative comments?

**Comment by Quasilyte** [comment]
@fexolm check top-level declarations doc comments.
They are stored inside AST nodes.
It's worthwhile to inspect go lint sources to find which comments it does handle, and trigger only for these. Our main goal is to detect when someone tries to silence go lint.
Because it's will be possible to disable this particular checker in future, people may continue to silence go lint this way, but at least there is an option to quickly find these spots when it's refactoring time,
